photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriver.scala

```diff
@@ -639,7 +638,7 @@ object GameTrainingDriver extends GameDriver {
 
         val searcher = getOrDefault(hyperParameterTuning) match {
           case HyperparameterTuningMode.BAYESIAN =>
-            new GaussianProcessSearch[GameEstimator.GameResult](dimension, evaluationFunction, evaluator)
+            new GaussianProcessSearch[GameEstimator.GameResult](dimension, evaluationFunction, isOptMax)
```

Comment:
@cmjiang I think it would be better / simpler to make the change in EvaluationFunction instead of having the system be aware of minimization vs maximization in all these places. Let's chat about it when I'm there in person next week.